code/game/objects/items/stacks/tiles/tile_types.dm

```diff
@@ -76,6 +76,17 @@
 	turf_type = /turf/open/floor/grass
 	resistance_flags = FLAMMABLE
 
+/obj/item/stack/tile/grass/attackby(obj/item/W, mob/user, params)
+	if((W.tool_behaviour == TOOL_SHOVEL) && params)
```

Comment:
why && params?

Reply:
I have copied the exact same behavior from how a shovel is applied to a grass turf, I do not know why it was used there.